src/Components/Menu.jsx

```diff
@@ -1,11 +1,16 @@
-import React from 'react'
+import React, { useContext } from 'react'
+import { QuizContext } from '../Helpers/Context'
+import "../Styles/App.css"
```

Comment:
no need to import styles in every file if its imported in root 
_remove this import_

Reply:
I have changed as you requested but I will commit all changes once I am done with all requests. rn playing game and working as well